### ISSUE TITLE
Bug on chat command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#89]
```
What steps will reproduce the problem?
1. send command "chat:florentl"
2. chatroom opens with florentl, write text
3. another chat room request is open with Florent Lopez

What is the expected output? What do you see instead?
There is multiple chat room for the same contact.

Please use labels and text to provide additional information.
Open chatroom (on chat command) with the contact full name

```

Original issue reported on code.google.com by `Florent....@gmail.com` on 8 Feb 2011 at 11:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 9 Feb 2011 at 8:01
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Didn't hat the time to test it, but this should fix it.
```

Original comment by `fschm...@gmail.com` on 9 Feb 2011 at 8:01

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9.1
```

Original comment by `Florent....@gmail.com` on 13 Feb 2011 at 12:48
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
